models/spring-ai-bedrock/src/main/java/org/springframework/ai/bedrock/jurassic2/BedrockAi21Jurassic2ChatClient.java

```diff
@@ -0,0 +1,107 @@
+package org.springframework.ai.bedrock.jurassic2;
```

Comment:
lost the license header, please check all files.

Reply:
@laglangyue  Fixed for all.